wp-content/themes/pub/wporg-learn-2024/patterns/archive-lessons-content.php

```diff
@@ -5,6 +5,8 @@
  * Inserter: no
  */
 
+get_the_excerpt();
```

Comment:
```suggestion
// Workaround for the first lesson in the grid not having an automatic excerpt generated
get_the_excerpt();
```

Reply:
Tested and this works for me, so I think it's good to merge as a workaround, with a comment. I think it would still be good to understand where this is failing though, and potentially raise a Gutenberg bug.

Reply:
2ebf0f6

Yeah, I spent some time looking into it, but couldn't find what part is causing the issue. There isn't any related discussion in Gutenberg either. Have already spent more time on this than expected, so just go with a workaround. I'll look into further the flow for this part in Gutenberg, and still can't figure it out, I'll open a ticket.